iidm/iidm-xml-converter/src/main/resources/xsd/iidm_V1_5.xsd

```diff
@@ -540,8 +547,10 @@
                 <xs:attribute name="g2_1" use="required" type="xs:double"/>
                 <xs:attribute name="b1_1" use="required" type="xs:double"/>
                 <xs:attribute name="b2_1" use="required" type="xs:double"/>
-                <xs:attribute name="xnodeP_1" use="required" type="xs:double"/>
-                <xs:attribute name="xnodeQ_1" use="required" type="xs:double"/>
+                <xs:attribute name="boundaryPointP_1" use="optional" type="xs:double"/>
+                <xs:attribute name="boundaryPointQ_1" use="optional" type="xs:double"/>
+                <xs:attribute name="boundaryPointV_1" use="optional" type="xs:double"/>
+                <xs:attribute name="boundaryPointAngle_1" use="optional" type="xs:double"/>
```

Comment:
Why don't use the same type for DanglingLine and TieLines and serialize them as a sub-element?